ConsoleApp_Task1/Program.cs

```diff
@@ -1,65 +1,17 @@
 ﻿using System;
 
 
-namespace ConsoleApp1_1
+namespace ConsoleApp_Task1
 {
     public class Figure
     {
-        public double input;
-
-        public Figure(double RadiusOrSide)
-        {
-            input = RadiusOrSide;
-        }
-
-        static double CalcCircle(double a)
-        {
-            const double Pi = Math.PI;
-            double result = Math.Round((a * a * Pi), 2);
-            return result;
-        }
-
-        static double CalcArea(double b)
-        {
-            double result = Math.Round(b * b, 2);
-            return result;
-        }
-
-        public static void Main(string[] args)
+        private double _input { get; set; }
+        public virtual double GetFigureArea()
         {
-            EnterFigure(Shape.Circle);
-            EnterFigure(Shape.Area);
-            Console.WriteLine("The end");
-            Console.ReadKey();
-        }
-
-        enum Shape
-        {
-            Circle,
-            Area
-        }
-
-        static void EnterFigure(Shape shape)
-        {
-            switch (shape)
-            {
-                case Shape.Circle:
-
-                    Console.WriteLine("What is the radius of the circle? Please populate and push Enter key \nNOTE:3 attempts are available otherwise random value will be taken");
-                    double MainResultCircle = CalcCircle(ReturnRadiusOrSide());
-                    Console.WriteLine("\n\rThe square of the circle is " + MainResultCircle);
-                    break;
-
-                case Shape.Area:
-
-                    Console.WriteLine("\nWhat is the side of the square? Please populate and push Enter key \nNOTE:3 attempts are available otherwise random value will be taken");
-                    double MainResultArea = CalcArea(ReturnRadiusOrSide());
-                    Console.WriteLine("\n\rThe square of the area is " + MainResultArea);
-                    break;
-            }
+            return 0;
         }
-
-        public static double ReturnRadiusOrSide()
+        public Figure() { }
+        public double GetValidatedInput()
         {
             int e = 0;
             while (e < 3)
```

Comment:
could you please use a constant in this task? maybe 3 value is a good candidate for it?